### ISSUE TITLE
space-around-operator warns about Unicode ranges.

### ISSUE THREAD

**Issue by bquorning** [sasstools/sass-lint#619]
Given this input:
```scss
@font-face {
  font-family: "Proxima Nova";
  unicode-range: U+1F00-1FFF;
}
```
sass-lint gives me the warning `Space expected around operator [Warning/space-around-operator]`. Apparently the `U+` prefix isn’t recognized.

**Comment by bgriffith** [comment]
Thanks - confirmed! Will look into it.

**Comment by bquorning** [comment]
Sounds good; thanks.
